src/EditorFeatures/VisualBasic/AutomaticEndConstructCorrection/AutomaticEndConstructCorrector.Session.vb

```diff
@@ -4,15 +4,16 @@ Imports Microsoft.CodeAnalysis.Editor.Shared.Utilities
 Imports Microsoft.VisualStudio.Text
 
 Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.AutomaticEndConstructCorrection
-    Partial Friend Class AutomaticEndConstructCorrector
+    Partial Friend Class AbstractCorrector
         Private Class Session
             Private ReadOnly _subjectBuffer As ITextBuffer
+            Private ReadOnly _shouldReplaceText As Func(Of String, Boolean)
```

Comment:
Rather than creating a lambda, just pass the AbstractCorrector? It'll be handy the next time you need something to go through.